bots/create-and-run-a-bot/rasa-bot/README.md

```diff
@@ -0,0 +1,104 @@
+# Run a Rasa Bot
+
+Rasa is the leading open-source machine learning toolkit that lets developers expand bots beyond answering simple questions
+with minimal training data. At the core, Rasa bot has a machine learning model which trained on example conversations.
```

Comment:
This was the original:
The bots are based on a machine learning model trained on example conversations.

One of these two:
Rasa bot has a machine learning model which "has been" trained on trained on example conversations
Rasa bot has a machine learning model trained on example conversations